src/settings/MatchProfiles/ViewMatchProfile.js

```diff
@@ -298,17 +278,23 @@ export class ViewMatchProfile extends Component {
       },
     };
 
-    return (
-      <Pane
+    const header = renderProps => (
```

Comment:
Move this function to the class methods and rename it, please

Reply:
Done!